addon/components/cfb-form-editor/question.js

```diff
@@ -42,6 +42,7 @@ export default Component.extend(ComponentQueryManager, {
 
   notification: service(),
   intl: service(),
+  options: service(),
```

Comment:
Maybe you could simplify your code by adding
```
namespace: reads("options.namespace")
```
(didn't test if that works with properties on services, but i guess it should)

Reply:
Interesting! I'll try that :-)